gflib/sprite.c

```diff
@@ -280,12 +276,16 @@ u32 gOamMatrixAllocBitmap;
 u8 gReservedSpritePaletteCount;
 
 EWRAM_DATA struct Sprite gSprites[MAX_SPRITES + 1] = {0};
-EWRAM_DATA static u16 sSpritePriorities[MAX_SPRITES] = {0};
 EWRAM_DATA static u8 sSpriteOrder[MAX_SPRITES] = {0};
+#if TESTING
+EWRAM_DATA bool8 sShouldProcessSpriteCopyRequests = 0;
+#else
 EWRAM_DATA static bool8 sShouldProcessSpriteCopyRequests = 0;
+#endif
```

Comment:
I get that this is exposed here for the tests, but my thoughts are as follows:

If this is necessary for tests to work, just make it global ( ? ) - This just seems overly verbose :D 
However as I see it the exposure is only to include setting this variable in the tests? If so for a performance benchmark I think this can be neglected, or substituted for setting some "dummy" variable if you really think it will have a noticable impact that one global is set :)

Reply:
You know what, you're totally right. I think I got a bit tunnel visioned—will remove the `TESTING` stuff from `gflib` :)